### PR TITLE
New FAQ for install index page

### PR DESCRIPTION
**Pull request by dbradish-microsoft**

### CODE REVIEW

**Comment by dbradish-microsoft** [comment]
@bebound, @jiasli  passed this PR review to you.

File: docs-ref-conceptual/install-azure-cli.md [hunk 1]
```diff
@@ -35,6 +35,31 @@ The Azure CLI is available to install in Windows, macOS and Linux environments.
 > [!NOTE]
 > If you're using the Azure classic deployment model, [install the Azure classic CLI](install-classic-cli.md).
 
+## FAQ
+
+### Where is the Azure CLI installed?
+
+When installing the Azure CLI, you cannot select an install location. In Windows, the 32-bit Azure CLI installs in `C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2` and the 64-bit in `C:\Program Files\Microsoft SDKs\Azure\CLI2`. In Linux, programs are more distributed.  Most binaries are in `/bin` or `/sbin` and most libraries are in `/lib`.
+
+User-specific configuration files are located in `$HOME/.azure` on macOS and Linux, and `%USERPROFILE%\.azure` on Windows.  These locations are known as the `AZURE_CONFIG_DIR`.
```
Comment (bebound) [inline docs-ref-conceptual/install-azure-cli.md:44]:
```suggestion
User-specific configuration files are located in `$HOME/.azure` on macOS and Linux, and `%USERPROFILE%\.azure` on Windows. These locations are known as the `AZURE_CONFIG_DIR`.
```

File: docs-ref-conceptual/install-azure-cli.md [hunk 2]
```diff
@@ -35,6 +35,31 @@ The Azure CLI is available to install in Windows, macOS and Linux environments.
 > [!NOTE]
 > If you're using the Azure classic deployment model, [install the Azure classic CLI](install-classic-cli.md).
 
+## FAQ
+
+### Where is the Azure CLI installed?
+
+When installing the Azure CLI, you cannot select an install location. In Windows, the 32-bit Azure CLI installs in `C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2` and the 64-bit in `C:\Program Files\Microsoft SDKs\Azure\CLI2`. In Linux, programs are more distributed.  Most binaries are in `/bin` or `/sbin` and most libraries are in `/lib`.
```
Comment (bebound) [inline docs-ref-conceptual/install-azure-cli.md:42]:
```suggestion
When installing the Azure CLI, you cannot select an install location. In Windows, the 32-bit Azure CLI installs in `C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2` and the 64-bit in `C:\Program Files\Microsoft SDKs\Azure\CLI2`. In Linux, programs are more distributed. It's located in `/opt/az/` on Ubuntu and Debian,  `/lib64/az/` on CentOS, RHEL and Azure Linux
```

Comment (dbradish-microsoft) [reply docs-ref-conceptual/install-azure-cli.md]: Thanks, @bebound!  I've added this information.

**Review by bebound — APPROVED** [review]

**Comment by dbradish-microsoft** [comment]
Merging w/ approval from @bebound.

**Pull request merged**